src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
+const sqsClient = new SQSClient({
+  region: REGION,
+  credentials: {
+    accessKeyId: process.env.VUE_APP_AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.VUE_APP_AWS_SECRET_KEY,
+  },
+});
+
+async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userID, doesUserExist, authType) {
+    const messageBody = [
+      {
+        dateTime: Date.now().toString(),
```

Comment:
What format does the `dateTime` get sent in?

Reply:
String. Its stored in BQ as string too.

Reply:
Haan that is clear. I meant what is the format of the date string?

Reply:
It was milliseconds before, now changed it to a proper timestamp format: '2021-09-14T10:16:29.779Z'